### PR TITLE
Add op csv tests to script_tests.json

### PR DESCRIPTION
**Pull request by Christewart**
Adds OP_CHECKSEQUENCEVERIFY tests to script_tests.json

### CODE REVIEW

File: src/script/script.cpp [hunk 1]
```diff
@@ -132,7 +132,8 @@ const char* GetOpName(opcodetype opcode)
     // expanson
     case OP_NOP1                   : return "OP_NOP1";
     case OP_CHECKLOCKTIMEVERIFY    : return "OP_CHECKLOCKTIMEVERIFY";
-    case OP_NOP3                   : return "OP_NOP3";
+    case OP_CHECKSEQUENCEVERIFY    : return "OP_CHECKSEQUENCEVERIFY"; 
+    //case OP_NOP3                   : return "OP_NOP3";
```
Comment (instagibbs) [inline src/script/script.cpp]: delete this line

**Comment by btcdrak** [comment]
You can add extra tests, as was commented on #7540, renaming isn't appropriate until the softfork activated, since it's still a NOP at the moment.

File: src/script/script.cpp [hunk 2]
```diff
@@ -132,7 +132,7 @@ const char* GetOpName(opcodetype opcode)
     // expanson
     case OP_NOP1                   : return "OP_NOP1";
     case OP_CHECKLOCKTIMEVERIFY    : return "OP_CHECKLOCKTIMEVERIFY";
-    case OP_NOP3                   : return "OP_NOP3";
+    case OP_NOP3                   : return "OP_NOP3";
```
Comment (fanquake) [inline src/script/script.cpp]: Looks like you've added a space to the end of this line?

File: src/test/data/script_tests.json [hunk 3]
```diff
@@ -232,8 +232,8 @@
 ["'abcdefghijklmnopqrstuvwxyz'", "HASH256 0x4c 0x20 0xca139bc10c2f660da42666f72e89a225936fc60f193c161124a672050c434671 EQUAL", "P2SH,STRICTENC", "OK"],
 
 
-["1","NOP1 CHECKLOCKTIMEVERIFY NOP3 NOP4 NOP5 NOP6 NOP7 NOP8 NOP9 NOP10 1 EQUAL", "P2SH,STRICTENC", "OK"],
-["'NOP_1_to_10' NOP1 CHECKLOCKTIMEVERIFY NOP3 NOP4 NOP5 NOP6 NOP7 NOP8 NOP9 NOP10","'NOP_1_to_10' EQUAL", "P2SH,STRICTENC", "OK"],
+["1", "NOP1 CHECKLOCKTIMEVERIFY NOP3 NOP4 NOP5 NOP6 NOP7 NOP8 NOP9 NOP10 1 EQUAL", "P2SH,STRICTENC", "OK"],
```
Comment (instagibbs) [inline src/test/data/script_tests.json]: more extra spaces here and one line below

**Comment by laanwj** [comment]
ACK after squash

**Comment by MarcoFalke** [comment]
@Christewart There is a tutorial written on how to squash, if you need help: https://github.com/fanquake/bitcoin/blob/90963e53707cf244e6a2a804341e9280628b0c34/CONTRIBUTING.md#squashing-commits

**Pull request merged**